### DISCUSSION TITLE
PSYCHOCLOWN NPC DOES NOT HAVE TALENTS IN GAME (THIS IS A BUG)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14018] [Bug reports]
### DISCLAIMERS

- [X] I HAVE SEARCHED THE ISSUE TRACKER TO CHECK IF THE ISSUE HAS ALREADY BEEN REPORTED.
- [ ] MY ISSUE HAPPENED WHILE USING MODS.

### WHAT HAPPENED?

PSYCHOCLOWN NPC DOES NOT HAVE TALENTS 

THE EVENT GIVES IT TALENTS

![333884773-089a00cb-7ef5-429d-bf3b-fe9e87657844](https://github.com/FakeFishGames/Barotrauma/assets/73229309/f9be3dca-5408-42b5-95bd-17f9b1f3b580)

BUT IN GAME, PSYCHOCLOWN NPC DOES NOT HAVE TALENTS 

THIS IS A BUG

### REPRODUCTION STEPS

    LOAD UP A LEVEL WITH BEACON
    TRIGGEREVENT BEACONCLOWN
    CONTROL THE CLOWN
    BEHOLD THE LACK OF TALENTS
    GIVETALENT "ENROLL INTO CLOWN COLLEGE"
    CLOWN POWER ACTIVATES

### VERSION

V1.4.6.0 (BLOOD IN THE WATER UPDATE, HOTFIX 2)

**Comment by Regalis11** [comment]
Ah, thank you for the extra info. I wasn't aware the event is supposed to give the character talents, so I thought [the previous report](https://github.com/FakeFishGames/Barotrauma/discussions/14004) was about you assuming the character should have the clown talents unlocked. Reopening the previous report and closing this one as a duplicate.
